### ISSUE TITLE
Text colour on some pages fails contrast requirements for accessibility

### ISSUE THREAD

**Issue by rogerhutchings** [zooniverse/Panoptes-Front-End#3346]
Spotted this while reviewing #3343. The body text colour on our static pages (e.g. `/about`) is #989898 against white. WCAG 2.0 requires [a minimum contrast ratio of 4.5:1](https://www.w3.org/TR/UNDERSTANDING-WCAG20/visual-audio-contrast-contrast.html), and ours fails that according to [this contrast calculator](http://leaverou.github.io/contrast-ratio/#%23989898-on-white).

**Comment by eatyourgreens** [comment]
See also https://www.paciellogroup.com/resources/contrastanalyser/
